test/integration/targets/filter_urls/runme.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eux
+
+export ANSIBLE_ROLES_PATH=../
+
+ansible-playbook -i ../../inventory runme.yml "$@"
```

Comment:
No need for inventory for this test since filters only run on the controller:

```suggestion
ansible-playbook runme.yml "$@"
```